images/kcp-registrar/register.sh

```diff
@@ -219,6 +219,10 @@ register_cluster() {
             --resources "$CRS_TO_SYNC"\
             --output-file "$syncer_manifest"
         add_ca_cert_to_syncer_manifest "${kcp_kcfg}" "$syncer_manifest"
+        # Add annotations required by kcp-glbc
+        KUBECONFIG="${kcp_kcfg}" kubectl annotate --overwrite synctarget "${sync_target_name}" featuregates.experimental.workload.kcp.dev/advancedscheduling='true'
```

Comment:
I suppose this annotation will become obsolete at some point. Is there a specific issue we should be looking at? If so please add it as a comment so we get a chance to clean it.

Reply:
There may be in the future support for multiple schedulers. Annotations may be used for selection. Some of them may be behind a feature gate others not. glbc will surely support something that is out of "experimental" status. There is work currently done on the scheduling side, especially in regards to what we discussed yesterday that would be useful for the gateway: syncing the Repository resources on multiple compute clusters. [Here is the epic](https://github.com/kcp-dev/kcp/issues/1898). I am not aware of a specific story looking at graduating advanced scheduling. It may be a bit early.